src/Backups/BackupCoordinationStage.h

```diff
@@ -43,6 +43,10 @@ namespace BackupCoordinationStage
 
     /// Coordination stage meaning that a host finished its work.
     constexpr const char * COMPLETED = "completed";
+
+    /// Coordination stage meaning that backup/restore has failed due to an error
+    /// Check '/error' for the error message
+    constexpr const char * ERROR = "error";
 }
 
 }
```

Comment:
For consistency it seems it's better to set the stage `SCHEDULED_TO_START` via `backup_coordination->setStage()` like we do it with other stages instead of calling

```
zk->trySet(path, Stage::SCHEDULED_TO_START, stat.version)
```
in `hasConcurrentRestores()`.

Reply:
I used `trySet` here because I want to verify the `stat.version ` as well (to confirm no other nodes were added during concurrency check)